src/styles/main.scss

```diff
@@ -1,3 +1,75 @@
+$weekdays: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6
+);
+$month-lengths: 28, 29, 30, 31;
+$calendar-day-gap: 1px;
+$calendar-day-width: 100px - $calendar-day-gap;
+
 body {
+  height: 100vh;
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 712px;
+  gap: 1px;
+  padding: 10px;
+}
+
+@each $day, $number in $weekdays {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: $number * ($calendar-day-width);
+    }
+  }
+}
+
+
+@for $calendar__day from 1 to 32 {
+  .calendar__day:nth-child(#{$calendar__day}) {
+    flex: 0 0 calc($calendar-day-width - $calendar-day-gap);
+    box-sizing: border-box;
+    border: 1px solid black;
+    width: 100px;
```

Comment:
Create the variable for this size

Reply:
https://github.com/mate-academy/layout_calendar/pull/3038/files#r1446322704
need fix it